config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml

```diff
@@ -9,7 +9,7 @@ spec:
       node.kubernetes.io/worker: ""
   config:
     installType: bundle
-    payloadImage: quay.io/confidential-containers/runtime-payload-ci:enclave-cc-HW-cc-kbc-latest
+    payloadImage: quay.io/confidential-containers/runtime-payload-ci:enclave-cc-HW-cc-kbc-v0.9.1
```

Comment:
I think this needs to change from `runtime-payload-ci` to `runtime-payload`

Reply:
ohh great catch; thank you

Reply:
Very nice catch, indeed!

```
@github-actions
enclave-cc e2e tests / operator enclave-cc e2e tests (pull_request) Successful in 5m
```